core/templates/dev/head/pages/story-editor-page/editor-tab/story-node-editor.directive.html

```diff
@@ -28,6 +28,10 @@
   <form class="form-horizontal" role="form" ng-submit="updateExplorationId(explorationId)">
     <input class="protractor-test-exploration-id-input" type="text" ng-model="explorationId" ng-change="checkCanSaveExpId()">
     <button type="submit" class="protractor-test-exploration-id-save-button btn btn-success btn-sm" ng-disabled="!canSaveExpId || (explorationId === currentExplorationId)">Save</button>
+    <button type="button" ng-click="removeExploration()" class="protractor-test-exploration-id-save-button btn btn-success btn-sm">Remove Exploration</button>
+    <span class="form-text error-message" ng-if="!expIdIsValid">
+      <em>Please enter a valid exploration id.</em>
```

Comment:
Can you make sure this message is removed when the user starts editing the field again?
Basically if an ID is invalid, this message is shown, but once user starts editing it, remove this.

Reply:
Done